account_invoice_show_currency_rate/models/account_move.py

```diff
@@ -32,6 +32,10 @@ def _compute_invoice_currency_rate(self):
             )
         return res
 
+    @api.depends("state", "line_ids.amount_currency", "line_ids.balance")
```

Comment:
Please put here a comment explaining the same you have put in the PR comment. If not, in the future, someone looking to this will be tempted to remove it as it's not deduced easily why this is here.